example_9/bringup/launch/rrbot_gazebo_sim.launch.xml

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright (c) 2024, Stogl Robotics Consulting UG (haftungsbeschränkt)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+
+Source of this file are templates in [RosTeamWorkspace](https://github.com/StoglRobotics/ros_team_workspace) repository.
+
+Author: Dr. Denis
+-->
+
+<launch>
+  <arg name="gui"
+       default="false"
+       description="Start RViz2 automatically with this launch file."/>
+
+  <!-- Load description and start controllers -->
+  <let name="robot_description_content" value="$(command '$(find-exec xacro) $(find-pkg-share ros2_control_demo_example_9)/urdf/rrbot.urdf.xacro use_gazebo_sim:=true')" />
+
+  <node pkg="robot_state_publisher" exec="robot_state_publisher" output="both">
+    <param name="robot_description" value="$(var robot_description_content)" />
+    <param name="use_sim_time" value="true" />
+  </node>
+
+  <!-- Start Gazebo -->
+  <let name="gazebo_world" value="$(find-pkg-share ros2_control_demo_example_9)/worlds/empty.world" />
+  <include file="$(find-pkg-share ros_gz_sim)/launch/gz_sim.launch.py" >
+    <arg name="gz_args" value=" -r -v 4 $(var gazebo_world)"/>
+  </include>
+
+  <node pkg="ros_gz_bridge" exec="parameter_bridge" args="
+        /clock@rosgraph_msgs/msg/Clock[ignition.msgs.Clock"
+        output="screen">
+  </node>
+
+  <!-- Create robot in Gazebo -->
+  <node pkg="ros_gz_sim" exec="create" output="screen" args="-topic /robot_description"/>
+
+  <!-- Start Controllers -->
+  <!--joint_state_braodcaster_spawner-->
+  <node pkg="controller_manager" exec="spawner" args="joint_state_broadcaster">
+    <param name="use_sim_time" value="true" />
+  </node>
+  <!--Mobile Base Controller-->
+  <node pkg="controller_manager" exec="spawner" args="forward_position_controller">
+    <param name="use_sim_time" value="true" />
+  </node>
+
+  <!-- Rviz -->
+  <node if="$(var gui)" pkg="rviz2" exec="rviz2" output="log" args="-d $(find-pkg-share $(var ros2_control_demo_description))/rrbot/rviz/rrbot.rviz">
+    <param name="use_sim_time" value="true" />
+  </node>
+
+</launch>
```

Comment:
Dear @destogl maybe we do not need the equivalent xml file here, as it is not in the other examples.